### PR TITLE
add

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

**Review by ghfvs-tester — APPROVED** [review]
lgmt

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:11]: review

Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:11]: ldksksk

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTestProject1
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: cool test

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/UnitTest1.cs]: some reply

**Review by meaghanlewis — COMMENTED** [review]
aaaa

**Review by meaghanlewis — COMMENTED** [review]
lalalalala

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTestProject1
+{
+    [TestClass]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs:6]: lalalalala

**Pull request merged**